logic/standalone/conflict.py

```diff
@@ -0,0 +1,209 @@
+'''
+<SUMMARY TBA>
+
+USAGE EXAMPLE:
+    raw_dict = conflict.CheckConflicts(playdo, _LIST_LIGHTING_OBJ)
+    pruned_dict = conflict.PruneConflicts(playdo, conflict_dictionary)
+    conflict.FixConflicts(playdo, pruned_dict)
+'''
+
+import logic.common.log_utils as log
+import logic.common.tiled_utils as tiled_utils
+
+#--------------------------------------------------#
+'''Variables'''
+
+
+
+
+
+#--------------------------------------------------#
+'''Public Functions'''
+
+def CheckConflicts(playdo, list_scan_obj):
+	''' Task 1 - Check if there is any conflict '''
+	log.Info("\n--- Checking for sort value conflicts ---\n")
+
+	# Create the dictionary with each unique sort value as keys
+	dict = {}
+	for scanned_obj in list_scan_obj:
+		curr_obj_list = playdo.GetAllObjectsWithName(scanned_obj)
+		log.Info( f'# of \"{scanned_obj}\": {len(curr_obj_list)}' )
+		for obj in curr_obj_list:
+			curr_string = tiled_utils.GetSort(obj)
+			if curr_string == None: continue
+			if not curr_string in dict:
+				dict[curr_string] = []
+			dict[curr_string].append(obj)
+
+
+	# Print the result
+	max_name_len = len( max(list_scan_obj, key = len) )    # Length of longest name in list
+	log.Info('')
+	log.Info('~~Note: Sort Layers with only 1 element are skipped~~')
+	for key, value in dict.items():
```

Comment:
Name changed from `key` to `key_sort`, also `value` to `list_obj`.

Also you mentioned not needing to spend too much effort on the output message later since you likely wouldn't be looking at it after all. Want to confirm whether that's true, or whether you still want me to sort the itmes.